drivers/volume/kdmp/kdmp.go

```diff
@@ -154,15 +152,15 @@ func getProvisionerName(pvc v1.PersistentVolumeClaim) string {
 	return provisioner
 }
 
-func isCSISnapshotClassRequired(pvc *v1.PersistentVolumeClaim) bool {
+func isCSISnapshotClassRequired(pvc *v1.PersistentVolumeClaim) (string, bool) {
 	pv, err := core.Instance().GetPersistentVolume(pvc.Spec.VolumeName)
 	if err != nil {
 		errMsg := fmt.Sprintf("error getting pv %v for pvc %v: %v", pvc.Spec.VolumeName, pvc.Name, err)
 		logrus.Warnf("ifCSISnapshotSupported: %v", errMsg)
 		// In the case errors, we will allow the kdmp controller csi steps to decide on snapshot support.
-		return true
+		return "", true
```

Comment:
Does this check make it to take direct kdmp backup?

Reply:
Yes, If DE does not have SnapshotStorageClass it will be taken as direct KDMP